internal/syncd/integrations/k8s/informer_test.go

```diff
@@ -0,0 +1,111 @@
+package k8s
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	shipitv1beta1 "ship-it-operator/api/v1beta1"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/cache/informertest"
+)
+
+func newFakeCache() *informertest.FakeInformers {
+	scheme := runtime.NewScheme()
+	shipitv1beta1.AddToScheme(scheme)
+
+	return &informertest.FakeInformers{
+		Scheme: scheme,
+	}
+}
+
+func TestLookup(t *testing.T) {
+	helmReleaseKind := "HelmRelease"
+	wordCountsRelease := "word-counts-release"
+	wordCountsRepository := "word-counts-repo"
+
+	fakeCache := newFakeCache()
+
+	fakeInformer, err := fakeCache.FakeInformerForKind(shipitv1beta1.Kind(helmReleaseKind))
+	if !assert.NoError(t, err) {
+		t.FailNow()
+	}
+
+	informer, err := NewInformerWithCache(context.Background(), fakeCache)
+	if !assert.NoError(t, err) {
+		t.FailNow()
+	}
+
+	values := map[string]interface{}{
+		"image": map[string]interface{}{
+			"repository": wordCountsRepository,
+			"tag":        "foo",
+		},
+	}
+
+	valuesRaw, err := json.Marshal(values)
+	if !assert.NoError(t, err) {
```

Comment:
Why not just `assert.NoError(t, err)`? I recall commenting this on another PR, but I believe it was merged before I saw the reason.

Reply:
https://github.com/Wattpad/ship-it/pull/50#discussion_r304529052

Reply:
Makes sense! My bad.